microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -544,6 +544,17 @@ def _install_dependencies(self) -> None:
 
         if isinstance(node.os, Ubuntu):
             node.os.add_repository("ppa:canonical-server/server-backports")
+            if "14.04" in node.os.information.release:
```

Comment:
Use `node.os.information.version <= "14.04.0"`, it matches more than the string.